### PR TITLE
Reorganise docs

### PR DESCRIPTION
**Pull request by layday**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -41,9 +41,9 @@ console_scripts =
     pyproject-build = build.__main__:entrypoint
 
 [options.extras_require]
-doc =
+docs =
```
Comment (FFY00) [inline setup.cfg:44]: Why?

**Review by FFY00 — APPROVED** [review]
It's fine, although I'd prefer to have the changes in separate commits. If you don't want to do that, don't let it be a blocker.

**Review by gaborbernat — APPROVED** [review]

File: setup.cfg (hunk 1 above)
Comment (layday) [reply setup.cfg:44]: I suppose because we speak of "docs" in the commit messages and the folder itself is called "docs".

Comment (layday) [reply setup.cfg:44]: (I won't split the commits up if that's ok but I'll keep it in mind for future PRs.)

Comment (FFY00) [reply setup.cfg:44]: Yeah, that's fine.

**Pull request merged**